charts/kubernetes-mcp-server/values.yaml

```diff
@@ -0,0 +1,121 @@
+# -- Enable OpenShift specific features
+openshift: false
+
+# -- This will set the replicaset count more information can be found here: https://kubernetes.io/docs/concepts/workloads/controllers/replicaset/
+replicaCount: 1
+
+# -- This sets the container image more information can be found here: https://kubernetes.io/docs/concepts/containers/images/
+image:
+  registry: quay.io
+  repository: manusa/kubernetes_mcp_server
+  # -- This sets the tag or sha digest for the image.
+  version: latest
```

Comment:
should this go to `latest`? or do we want to pin it to the most recent release?

Reply:
I think generally, we should have image somewhere else published, instead of `manusa` repo :sweat_smile: